rules/do-you-zz-old-files-rather-than-deleting-them/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you 'zz' old files rather than deleting them?
-uri: do-you-zz-old-files-rather-than-deleting-them
+uri: zz-files
```

Comment:
1. Please rename the folder to this new uri
2. Add `do-you-zz-old-files-rather-than-deleting-them` as a redirect 
3. Find all categories that reference `do-you-zz-old-files-rather-than-deleting-them` and update them to `zz-files`

Reply:
Done - please check the changes and commit 😊